src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Models/LayoutSettings.cs

```diff
@@ -18,7 +18,7 @@ public class LayoutSettings
         public Components Components { get; set; }
     }
 
-    /// <summary>
+     /// <summary>
     /// Pages
     /// </summary>
     public class Pages
```

Comment:
Multiple classes in the same file. Is that ok?

Reply:
Spittet old and new classes in seperate files.